src/app/contribute/contribute-action-button.component.ts

```diff
@@ -7,16 +7,16 @@ import { Component } from '@angular/core';
 export class ContributeActionButtonsComponent {
   public routes = [
     {
-      name: 'Overview',
+      name: 'Contribution process',
```

Comment:
the route for this is still `https://developer.blackbaud.com/skyux/contribute/overview`, do we want to change that to `https://developer.blackbaud.com/skyux/contribute/process` or something?

Reply:
I don't have a strong preference, but it did occur to me that maybe we should change this. Since you seem to have the same thought, I'll go ahead and update the path and all the related links.